### ISSUE TITLE
Error: Geo nodes should have only one other edge

### ISSUE THREAD

**Issue by antonykamp** [simulate-digital-rail/orm-importer#47]
Hey guys 👋 

I want to import the following polygon:
```
51.53832786538389 14.254009723663332 51.53616568452972 14.256412982940676 51.536833035258354 14.262421131134035 51.53934218643719 14.261519908905031
```

But sadly, I got the following error
 
```
  File "/Users/antonykamp/Projects/hpi-bp/yaramo/demo/app.py", line 19, in run_converter
    topology = ORMImporter().run(polygon)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/antonykamp/Library/Caches/pypoetry/virtualenvs/demo-QQ3WmLyZ-py3.11/lib/python3.11/site-packages/orm_importer/importer.py", line 123, in run
    next_top_node, path = self._get_next_top_node(node, edge, [])
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/antonykamp/Library/Caches/pypoetry/virtualenvs/demo-QQ3WmLyZ-py3.11/lib/python3.11/site-packages/orm_importer/importer.py", line 64, in _get_next_top_node
    raise Exception(f"Node: {node_to_id}. \n Geo nodes should have only one other edge, otherwise we don't know where to go")
Exception: Node: 9308323907.
 Geo nodes should have only one other edge, otherwise we don't know where to go
```

**Comment by antonykamp** [comment]
Here is an even smaller testset:
```
51.537036575283324 14.25584837794304 51.53704658542498 14.255620390176775 51.53692813194067 14.255167096853258 51.53687307599091 14.25525560975075 51.53682135670473 14.255505055189134 51.53703991199746 14.25616353750229 51.53709997281011 14.256085753440859
```

**Comment by jeriox** [comment]
> Hey guys 👋
> 
> I want to import the following polygon:
> 
> ```
> 51.53832786538389 14.254009723663332 51.53616568452972 14.256412982940676 51.536833035258354 14.262421131134035 51.53934218643719 14.261519908905031
> ```
> 
> But sadly, I got the following error
> 
> ```
>   File "/Users/antonykamp/Projects/hpi-bp/yaramo/demo/app.py", line 19, in run_converter
>     topology = ORMImporter().run(polygon)
>                ^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Users/antonykamp/Library/Caches/pypoetry/virtualenvs/demo-QQ3WmLyZ-py3.11/lib/python3.11/site-packages/orm_importer/importer.py", line 123, in run
>     next_top_node, path = self._get_next_top_node(node, edge, [])
>                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Users/antonykamp/Library/Caches/pypoetry/virtualenvs/demo-QQ3WmLyZ-py3.11/lib/python3.11/site-packages/orm_importer/importer.py", line 64, in _get_next_top_node
>     raise Exception(f"Node: {node_to_id}. \n Geo nodes should have only one other edge, otherwise we don't know where to go")
> Exception: Node: 9308323907.
>  Geo nodes should have only one other edge, otherwise we don't know where to go
> ```

This should be fixed by #44. Although I tested your original polygon locally and it crashes on node 9309173893. As far as I see, this is due to an error in the data, as this node is not tagged as railway=switch and therefor ignored while building the topology.

**Comment by antonykamp** [comment]
Thank you for the quick investigation 👍 
Running the query in overpass turbo, I found the node, and it has the tag "railway=switch" 😕
```xml
<node id="9309173893" lat="51.5373402" lon="14.2571453">
    <tag k="railway" v="switch"/>
</node>
```

**Comment by jeriox** [comment]
sorry, wrong node id. 9308323907 is the node in question

**Comment by jeriox** [comment]
The original issue should be fixed by #44. If the issue persists after correcting the data in the node, please reopen the issue.
